trymito.io/styles/[slug].module.css

```diff
@@ -107,12 +183,189 @@
     border-radius: 1rem;
 }
 
-@media only screen and (min-width: 50em) {
+.post_content_container {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.post_content p, .post_content li, .post_content a {
+    font-size: 1.2rem;
+}
+
+.post_content table {
+    font-size: 1rem;
+}
+
+
+/* Post CTA and TOC */
+.post_toc {
+    width: 16rem;
+    min-width: 16rem;
+    height: fit-content;
+    position: sticky;
+    top: 5rem;
+    margin-left: 2rem;
+    margin-right: 2rem;
+}
+
+.post_cta {
+    width: 16rem;
+    min-width: 16rem;
+    height: fit-content;
+    position: sticky;
+    top: 6rem;
+    background-color: #F2C94C;
+    border-radius: 15px;
+    padding: 2rem 2rem 0 2rem;
+    margin-right: 2rem;
+    margin-top: 2rem;
+    margin-left: 2rem;
+}
+
+.post_cta a {
+    text-decoration: none;
+}
+
+/* Sharing Buttons */
+.share_section {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-direction: row;
+    margin-top: 2rem;
+}
+
+.share_section a {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 5px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-right: 0.5rem;
+}
+
+.share_section a:hover {
+    background-color: var(--color-light-background);
+}
+
+/* Suggested Posts */
+.suggested_posts_section {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    margin-top: 2rem;
+    max-width: 70rem;
+}
+
+.suggested_posts_container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    margin-bottom: 1rem;
+}
+
+.suggested_post {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    margin-right: 1rem;
+    text-decoration: none;
+    color: var(--color-text-primary);
+    width: calc(50% - 1rem);
```

Comment:
I think again we can just let the flex box handle this. Instead of specifying that you want the suggest post to be half the width and leave space for a margin, just set the margin you want and let the flex box figure out the width -- that's the beauty of flex boxes!